storm-webapp/src/main/java/org/apache/storm/daemon/ui/WEB-INF/templates/topology-page-template.html

```diff
@@ -74,12 +74,12 @@
             Assigned CPU (%)
           </span>
         </th>
-        {{/schedulerDisplayResource}}
         <th>
           <span data-toggle="tooltip" data-placement="top" title="Assigned Generic Rescources by Scheduler.">
             Assigned Generic Resources
           </span>
         </th>
+        {{/schedulerDisplayResource}}
```

Comment:
This makes table headers match the logic lines ~ 112-115: https://github.com/apache/storm/blob/8a00852f4b21797289eefa8c0b58b0e671990b6b/storm-webapp/src/main/java/org/apache/storm/daemon/ui/WEB-INF/templates/topology-page-template.html#L112-L115